templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig

```diff
@@ -1,5 +1,7 @@
 {% extends '@DemosPlanCore/DemosPlanCore/base_oeb.html.twig' %}
+
 {% if templateVars is defined %}
+
```

Comment:
I'd only use a blank line here if this section would be dense codewise.

Reply:
I learned that our convention is to put a blink line before every if condition

Reply:
before, but not after ;-)
```
const some  = code

if(some) {
  someCode()

  return foo
} else {
  return bar
}
```

Reply:
...unless the after is also a before:

```
{% if templateVars is defined %}

  {% if templateVars.title is defined %}
	 // something here
  {% endif %}

{% endif %}
```

Reply:
true